prac_05/state_names.py

```diff
@@ -5,14 +5,19 @@
 """
 
 # TODO: Reformat this file so the dictionary code follows PEP 8 convention
-CODE_TO_NAME = {"QLD":"Queensland", "NSW": "New South Wales", "NT" : "Northern Territory", "WA" : "Western Australia",
-            "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
 print(CODE_TO_NAME)
 
-state_code = input("Enter short state: ")
+for code, name in CODE_TO_NAME.items():
+    print(f"{code:<3} is {name}")
+
+
+state_code = input("Enter short state: ").upper()
```

Comment:
the function input doesnt need to be called twice

Reply:
bro,they are practicals 5, maybe check my files in practical 6 :)